evalme/image/object_detection.py

```diff
@@ -539,15 +539,8 @@ class BrushEvalItem(ObjectDetectionEvalItem):
     def _iou(gt, pred):
         gt = decode_rle(gt['rle'])
         pred = decode_rle(pred['rle'])
-        union = 0
-        intersection = 0
-        for item in zip(gt, pred):
-            if item[0] == item[1] and item[0] == 0:
-                pass
-            else:
-                union = union + 1
-                if item[0] == item[1] and item[1] != 0:
-                    intersection = intersection + 1
+        union = np.sum(np.logical_or(gt, pred).astype(np.int))
```

Comment:
why do we need to cast arrays to int? could we use `np.logical_or(gt, pred).sum()` instead, to avoid mem issues?